README.rst

```diff
@@ -17,7 +17,7 @@ VOC
 .. image:: https://img.shields.io/pypi/l/voc.svg
     :target: https://github.com/pybee/voc/blob/master/LICENSE
 
-.. image:: https://beekeeper.herokuapp.com/projects/pybee/voc/shield
+.. image:: https://img.shields.io/badge/beekeeper-voc-blueviolet.svg
```

Comment:
The correct link should be now https://beekeeper.beeware.org/projects/beeware/voc/shield